segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessorTest.java

```diff
@@ -598,6 +598,21 @@ public void testSegmentAttribute() throws Exception {
         order.verify(connection).send(new WireCommands.SegmentAttribute(9, WireCommands.NULL_ATTRIBUTE_VALUE));
     }
 
+    @Test(timeout = 200000)
+    public void testFlush() throws Exception {
```

Comment:
Same here, testing of this command should be in `AdminRquestProcessorImplTest`.

Reply:
Done!